pkg/plugins/policies/meshtimeout/plugin/v1alpha1/plugin_test.go

```diff
@@ -0,0 +1,485 @@
+					Configure(FilterChain(NewFilterChainBuilder(envoy_common.APIV3).
+						Configure(TcpProxy(
+							"127.0.0.1:10002",
+							envoy_common.NewCluster(
+								envoy_common.WithService("backend"),
+								envoy_common.WithWeight(100),
+							),
+						)),
+					)).
+					MustBuild(),
+			},
+				{
+					Name:     "outbound",
+					Origin:   generator.OriginOutbound,
+					Resource: clusterWithName("second-service"),
+				}},
+			toRules: core_xds.ToRules{
+				Rules: []*core_xds.Rule{
+					{
+						Subset: core_xds.Subset{core_xds.Tag{
+							Key:   mesh_proto.ServiceTag,
+							Value: "second-service",
+						}},
+						Conf: api.Conf{
+							ConnectionTimeout: parseDuration("10s"),
+							IdleTimeout:       parseDuration("30s"),
+						},
+					},
+				},
+			},
+			expectedCluster:  "basic_tcp_cluster.golden.yaml",
+			expectedListener: "basic_tcp_listener.golden.yaml",
+		}),
+		Entry("basic inbound route", sidecarTestCase{
+			resources: []core_xds.Resource{{
+				Name:     "inbound",
+				Origin:   generator.OriginInbound,
+				Resource: httpInboundListenerWith(80),
+			},
+				{
+					Name:     "inbound",
+					Origin:   generator.OriginInbound,
+					Resource: clusterWithName("localhost:80"),
+				}},
+			fromRules: core_xds.FromRules{
+				Rules: map[core_xds.InboundListener]core_xds.Rules{
+					{
+						Address: "127.0.0.1",
+						Port:    80,
+					}: []*core_xds.Rule{
+						{
+							Subset: core_xds.Subset{},
+							Conf: api.Conf{
+								ConnectionTimeout: parseDuration("10s"),
+								IdleTimeout:       parseDuration("1h"),
+								Http: &api.Http{
+									RequestTimeout:        parseDuration("5s"),
+									StreamIdleTimeout:     parseDuration("1s"),
+									MaxStreamDuration:     parseDuration("10m"),
+									MaxConnectionDuration: parseDuration("10m"),
+								},
+							},
+						},
+					}},
+			},
+			expectedCluster:  "basic_inbound_cluster.golden.yaml",
+			expectedListener: "basic_inbound_listener.golden.yaml",
+		}),
+		Entry("outbound with defaults when http conf missing", sidecarTestCase{
+			resources: []core_xds.Resource{{
+				Name:     "outbound",
+				Origin:   generator.OriginOutbound,
+				Resource: httpOutboundListenerWith(10001),
+			},
+				{
+					Name:     "outbound",
+					Origin:   generator.OriginOutbound,
+					Resource: clusterWithName("other-service"),
+				}},
+			toRules: core_xds.ToRules{
+				Rules: []*core_xds.Rule{
+					{
+						Subset: core_xds.Subset{
+							{
+								Key:   mesh_proto.ServiceTag,
+								Value: "other-service",
+							},
+						},
+						Conf: api.Conf{
+							ConnectionTimeout: parseDuration("10s"),
+							IdleTimeout:       parseDuration("1h"),
+						},
+					},
+				},
+			},
+			expectedCluster:  "outbound_with_defaults_cluster.golden.yaml",
+			expectedListener: "outbound_with_defaults_listener.golden.yaml",
+		}),
+	)
+
+	It("should generate proper Envoy config for MeshGateway Dataplanes",
+		func() {
```

Comment:
In all the test we keep the convention that `It()` and `func() {` goes in the same line

Reply:
fixed